packages/components/postcss.config.js

```diff
@@ -1,3 +1,9 @@
 module.exports = {
-  plugins: [require("tailwindcss"), require("autoprefixer")],
+  plugins: {
+    "postcss-import": {},
+    "tailwindcss/nesting": "postcss-nesting",
+    tailwindcss: {},
+    autoprefixer: {},
+    cssnano: {},
+  },
```

Comment:
A ticket is in the backlog to consolidate the config with the one used by storybook